api/discussion.adoc

```diff
@@ -39,6 +40,7 @@ Following outlines routes and endpoints for each that can be offered by Club-dat
 ====
 * Contributors
 * Created date
+* Updated date (with github hooks?)
```

Comment:
Can you please tell a little more on this one?

Reply:
To see if a project is dead.

Reply:
so getting the last commit date on the head of default branch